tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -556,7 +556,7 @@ def _pad_handler(self, trans, node):
             new_pads = [pads[0], pads[3], pads[1], pads[2], pads[4], pads[7], pads[5], pads[6]]
             node.set_attr("pads", new_pads)
             return self._switch_transpose_and_node(node, trans)
-        if node.inputs[1].is_const():
+        if node.inputs[1].is_const() and self._nodes_has_single_consumer_node([node.inputs[1]]):

```

Comment:
Not sure if this is masking the issue, or solving the issue.

Is it possible to add a UT that has  one Transpose feeding into 2  Pads followed by 2 transposes?  The transposes should be  set up such that they cancel out, leaving 0 transposes at the end of the optimization.


###Starting graph
```script
Input 1  --> Trans1 --> Pad 1 --> Trans2 --> Output1
                    --> Pad 2 --> Trans3 --> Output2
```
  

###Ending graph
```script
Input 1  --> Pad 1 --> Output1
         --> Pad 2 --> Output2
```

Reply:
Switching the approach - allowing it to continue no matter how many consumers there but keep the tensor from overly tranposed.